.buildkite/create_integration_testing_worksheet.py

```diff
@@ -0,0 +1,48 @@
+"""
+# Requirements:
+    * Generate a service account key for your Google API credentials, then create environment variable GOOGLE_APPLICATION_CREDENTIALS.
+        - e.g export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json.
+
+# Environment Variable/s:
+    * GOOGLE_APPLICATION_CREDENTIALS = Your service account key.
+"""
+
+from oauth2client.service_account import ServiceAccountCredentials
+import gspread
+import os
+
+
+WKS_CREDENTIALS = os.getenv("GOOGLE_APPLICATION_CREDENTIALS")
+WKS_TAG = os.getenv("BUILDKITE_TAG")
+
+WKS_TEMPLATE_TAG = 'release-v0.11.x'
+WKS_NAME = "Integration testing with Gherkin scenarios"
+WKS_INDEX_COPY = 0
+WKS_CELLS_RANGE = 100
+WKS_COLUMN = 'B'
+
+SCOPE = ['https://spreadsheets.google.com/feeds',
+         'https://www.googleapis.com/auth/drive']
+
+
+def copy_worksheet():
+	google_account = gspread.authorize(ServiceAccountCredentials.from_json_keyfile_name(WKS_CREDENTIALS, SCOPE))
+	wks = google_account.open(WKS_NAME)
+	wks_id = wks.get_worksheet(WKS_INDEX_COPY).id
+	return wks.duplicate_sheet(wks_id, insert_sheet_index=1, new_sheet_id=None, new_sheet_name=WKS_TAG)
```

Comment:
Having a sheet per test run all in a single document looks a bit unwieldy:

![image](https://user-images.githubusercontent.com/2367265/49303277-29414480-f47e-11e8-9ff2-44bcb74a89bc.png)

Have you guys (cc @radinamatic) considered alternate structures?

For example, one possibility might be a new document per release branch, and then having a sheet per test run? In other words, you could have a structure like:

* _[folder]_ Kolibri testing spreadsheets
  * _[spreadsheet]_ 0.10.x
    * _[sheet]_ 0.10.0-beta1
    * _[sheet]_ 0.10.0-beta2
    * _[sheet]_ 0.10.0
    * _[sheet]_ 0.10.1-beta1
    * _[sheet]_ ...
  * _[spreadsheet]_ 0.11.x
    * _[sheet]_ 0.11.0-dev-notifications
    * _[sheet]_ 0.11.0-beta1
    * ...

Also, just a heads-up that the spreadsheet currently exists in a KA Lite folder in addition to the Kolibri folder:

![ka](https://user-images.githubusercontent.com/2367265/49303738-580bea80-f47f-11e8-8ae1-8d38ef1c0cb3.png)